### PR TITLE
Update README.adoc

### PR DESCRIPTION
**Pull request by nathansit**
* Just fixed some typos
* Adjusted some wording
* Feel free to accept whichever parts make sense (if any)

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -24,8 +24,7 @@ This architecture brings the following advantages:
 * The JVM for running the actual builds does not need to get started anew for each build.
 * The classloaders holding classes of Maven plugins are cached over multiple builds. The plugin jars are thus read
   and parsed just once. SNAPSHOT versions of Maven plugins are not cached.
-* The native code produced by the Just-In-Time (JIT) compiler inside the JVM is kept too. Compared to stock Maven,
-  less time is spent by the JIT compilation. During the repeated builds the JIT-optimized code is available
+* The native code produced by the Just-In-Time (JIT) compiler inside the JVM is kept too. JIT compilation is faster than stock Maven. During the repeated builds the JIT-optimized code is available
```
Comment (ppalaga) [inline README.adoc:27]: > JIT compilation is faster than stock Maven.

This changes the meaning in a wrong way. The changed sentence does not state anything useful.

File: README.adoc [hunk 2]
```diff
@@ -70,7 +69,7 @@ $ brew install mvndaemon/homebrew-mvnd/mvnd
 
 We're looking for contribution to support https://www.macports.org[MacPorts],
 https://community.chocolatey.org/packages/mvndaemon/[Chocolatey], https://scoop.sh/[Scoop] or
-https://github.com/joschi/asdf-mvnd#install[asdf].  If you're fancy helping us...
+https://github.com/joschi/asdf-mvnd#install[asdf].  If you fancy helping us...
```
Comment (ppalaga) [inline README.adoc:72]: -1 on this change. The original was correct IMO.

**Review by ppalaga — CHANGES_REQUESTED** [review]

File: README.adoc (hunk 2 above)
Comment (Stephan202) [reply README.adoc:72]: I'm not a native speaker, but IIUC the original is incorrect, and the suggestion _is_ correct. See also examples e.g. [here](https://dictionary.cambridge.org/dictionary/english/fancy).

Comment (nathansit) [reply README.adoc:73]: I can see it being written as "If you are fancy, help us" or something along those lines. But I figured that the correction was your intended turn of phrase.

[Another example](https://hinative.com/questions/16176373) here the answers seem to change the you're to you

Comment (ppalaga) [reply README.adoc:73]: Thanks for pointing to an evidence, gentlemen. I did not know "fancy" was also a verb.

File: README.adoc [hunk 3]
```diff
@@ -183,7 +183,7 @@ Properties defined in the first files will take precedence over properties speci
 
 A few special properties do not follow the above mechanism:
 
-* `mvnd.daemonStorage`: this property defines the location where mvnd stores its files (registry and daemon logs).  This property can only be defined as a system property on the command line
+* `mvnd.daemonStorage`: this property defines the location where mvnd stores its files (registry and daemon logs).  This property can only be defined as a system property into the command line
```
Comment (Stephan202) [inline README.adoc:186]: I think that "on" is more idiomatic in this context :upside_down_face: (Not sure I've ever seen "into" used this way.)

**Review by Stephan202 — COMMENTED** [review]
The other two changes LGTM :)

File: README.adoc (hunk 3 above)
Comment (nathansit) [reply README.adoc:186]: Aha good catch I must have overwritten and then modified the wrong "on" when I force pushed.

**Review by ppalaga — APPROVED** [review]
Thanks  @nathansit!

**Pull request merged**